apispec/openapi.json

```diff
@@ -5518,17 +5671,26 @@
       "log.dateFrom": {
         "title": "datefrom",
         "type": "string",
-        "description": "Formatting: yyyyMMdd-HHmm, timezone UTC+0"
+        "description": "RFC3339 format date indicating the starting date.",
+        "x-examples": {
+          "example-1": "2013-01-01T22:30:00+08:00"
+        }
       },
       "log.dateTo": {
         "title": "dateto",
         "type": "string",
-        "description": "Formatting: yyyyMMdd-HHmm, timezone UTC+0"
+        "description": "RFC3339 format date indicating the ending date.\nIf the time range between datefrom and dateto is more than 366 days, 'TimeRangeTooLarge' error will be returned. ",
+        "x-examples": {
+          "example-1": "2013-01-01T22:30:00+08:00"
+        }
       },
       "log.logUrl": {
         "title": "log-url",
         "type": "string",
-        "description": "full URL of the log file that you can download.\nThe log file name format is\ndomain-port_YYYY-MM-DD-hhmm-hhmm_mproxy_X.log.gz and\ndomain-port_YYYY-MM-DD-hhmm-hhmm_mproxy_X.late.log.gz:\n1) domain-port represents the HDT transport;\n2) YYYY-MM-DD-hhmm-hhmm means the time range is from YYYY-MM-DD-hhmm to YYYY-MM-DD-hhmm;\n3) mproxy is a fixed string;\n4) late.log.gz refers to a log whose delay time is more than 2 hours. Usually, the proportion of the late log is small (no more than 1%.\n5). X is the sequence number. It increases when the uncompressed file size reaches 2GB(this is an approximation).\n"
+        "description": "URL of for downloading log file. The format of log file name are following:\ndomain-port_YYYY-MM-DD-hhmm-hhmm_mproxy_X.log.gz\ndomain-port_YYYY-MM-DD-hhmm-hhmm_mproxy_X.late.log.gz:\n1) domain-port represents the HDT transport;\n2) YYYY-MM-DD-hhmm-hhmm means the time range is from YYYY-MM-DD-hhmm to YYYY-MM-DD-hhmm;\n3) mproxy is a fixed string;\n4) late.log.gz refers to a log delay time is more than 2 hours. Usually, the proportion of the late log is small (no more than 1%).\n5) X is the sequence number. It increases when the uncompressed file size reaches 2GB.\n",
```

Comment:
Change "URL of for downloading log file" to "URL for downloading log file"
and change "The format of log file name are following" to
"The log file name follows this format:"